reference/format/search.rst

```diff
@@ -1,2 +1,137 @@
 Search Index
 ============
+
++-----------+------------------------------------------------------------+
+| Filename  | ``search.yml``                                             |
++-----------+------------------------------------------------------------+
+| Root Node | The fully qualified class name of the entity being indexed |
++-----------+------------------------------------------------------------+
+| Options   | * `alias`_                                                 |
+|           | * `fields`_                                                |
+|           |                                                            |
+|           |   * :ref:`name <reference-format-search-fields-name>`      |
+|           |   * `relation_fields`_                                     |
+|           |   * `relation_type`_                                       |
+|           |   * `target_fields`_                                       |
+|           |   * `target_type`_                                         |
+|           |                                                            |
+|           | * `label`_                                                 |
+|           | * `mode`_                                                  |
+|           | * `route`_                                                 |
+|           |                                                            |
+|           |   * :ref:`name <reference-format-search-route-name>`       |
+|           |   * `parameters`_                                          |
+|           |                                                            |
+|           | * `search_template`_                                       |
+|           | * `title_fields`_                                          |
++-----------+------------------------------------------------------------+
+
+``alias``
+---------
+
+**type**: ``string``
+
+An alias used in the ``from`` keyword in :ref:`advanced search <advanced-search-api>`.
+
+.. _reference-format-search-fields:
+
+``fields``
+----------
+
+**type**: ``sequence``
+
+The list of fields shown in the search result. Each entry must be a map that can use the following
```

Comment:
Fields does not shows in search result. This fields are `virtual fields` what you can use in search queries.